config/world_patrol/P05_WFLS/P05_WFLS_R09_QDGSCHZY_R03_GSCJ_2.yml

```
@@ -30,138 +30,163 @@ route:
+  - idx: 62
+    op: 'move'
+    data: [2623, 1184]
+  - idx: 63
+    op: 'gameplay_interact'
+    data: ['0']
+  - idx: 64
+    op: 'move'
+    data: [2689, 1121]
+  - idx: 65
+    op: 'patrol'
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)